examples/ubuntu-ui-toolkit-gallery/CMakeLists.txt

```diff
@@ -0,0 +1,89 @@
+cmake_minimum_required(VERSION 3.0.0)
```

Comment:
Is there some reason the existing qmake can't be used here? I don't think it makes sense to mix build systems, and introduce cmake just for building a click of the gallery, when there's already the `ubuntu-ui-toolkit-gallery-click.pro` file and clickable does support building qmake projects.

Reply:
humm, yes right, will take a look

Reply:
so finally did something, the only drawback is that we need to rename or remove locally the ubuntu-ui-toolkit-gallery.pro because clickable will read all `.pro` files otherwise even if i specified in clickable the `"build_args": ${SRC_DIR}/ubuntu-ui-toolkit-gallery-click.pro"`... Or if you know another way ?